assets/prototype/domain/eventEditor/tickets/ticketCard/EditTicketModal.js

```diff
@@ -6,8 +6,9 @@ import { useEntityMutator, EntityType } from '../../../../application/services/a
 import useDatetimes from '../../data/queries/datetimes/useDatetimes';
 
 const EditTicketModal = ({ relatedDates }) => {
-	const { id, getIsOpen, onClose } = useContext(TicketContext);
-	const isOpen = getIsOpen();
+	const { id, editorIds, editorState } = useContext(TicketContext);
```

Comment:
We should consider converting `useContext(TicketContext)` to something like `useTicket()` just the RFF `useForm()` and `useField()`.

Reply:
ya that's a good idea